tests/Extensions/Abstractions.UnitTests/Activities/ActivityExtensionsTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Omex.Extensions.Abstractions.Activities;
+using Microsoft.Omex.Extensions.Abstractions.Activities.Processing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Omex.Extensions.TimedScopes.UnitTests
+{
+	[TestClass]
```

Comment:
You could add a TestCategory("") in all test files.

Reply:
Makes sense, thank you, added